src/common/utils/Utils.ts

```diff
@@ -51,6 +51,17 @@ export function getArraySize(array: string[]): number {
   return 0;
 }
 
+export function traverseObjectRecursively(obj: any, callback: (visitedObject: any) => void): void {
```

Comment:
@ryanemerson It would, but how do I access object enclosing key/value pairs? I could not figure it out